### DISCUSSION TITLE
Materia base para T3

### DISCUSSION THREAD

**Discussion by elbosco** [IIC2133-PUC/2024-2#176] [Tarea 3] [answered]
Hola,
¿Qué materia teórica de las clases es necesaria para esta tarea?
Saludos,

**Comment by JavierLarre** [comment, marked as answer]
Algoritmos Greedy y Árboles de Cobertura Mínima.

**Reply by elbosco** [reply]
Hola, gracias por la respuesta. ¿Para los arboles de cobertura mínima no es necesario ver la clase de kruskal y prim o si?
Saludos,

**Reply by JavierLarre** [reply]
Si, es necesario.
